spk/salt-minion/src/conf/privilege

```diff
@@ -0,0 +1,31 @@
+{
+	"defaults":{
+		"run-as": "package"
+	},
+	"username": "sc-salt",
```

Comment:
Does it make sense to create a service user `sc-salt` if all scripts are run as `root`. Will `sc-salt` be used for anything relevant?
If not, you should simplify that file and use run-as root. For reference: https://help.synology.com/developer-guide/privilege/privilege_specification.html

Reply:
Ok. I will change it. It's the first time I use the new service framework, and I understood that it was recommended to keep each package owned by a user, even when run as root. I will push the change asap

Reply:
@kukoarmas In fact, you're right about ownership. "run-as: root" was a bad idea of mine to avoid listing all scripts in privilege file but it changes "everything" a wrong way. Sorry, may you please revert to first version.